src/test/java/com/wikia/webdriver/common/remote/operations/DiscussionsCategoryOperations.java

```diff
@@ -1,13 +1,17 @@
 package com.wikia.webdriver.common.remote.operations;
 
 import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
 import com.wikia.webdriver.common.remote.Discussions;
 import com.wikia.webdriver.common.remote.context.CategoryContext;
 import com.wikia.webdriver.common.remote.context.CreateCategoryContext;
 import com.wikia.webdriver.elements.mercury.components.discussions.common.category.CategoryPill;
 import lombok.AllArgsConstructor;
 import org.openqa.selenium.WebDriver;
 
+import java.rmi.RemoteException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.rmi.RemoteException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)